app/controllers/helps_controller.rb

```diff
@@ -5,6 +5,10 @@ def show
   end
 
   def oauth2
-    render layout: "empty"
+    if valid_config?
+      redirect_to root_path
+    else
+      render layout: "empty"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.